### PR TITLE
add current profile annotations to CVO manifests

### PR DESCRIPTION
**Pull request by guillaumerose**
This is matches openshift/enhancements#414 and doesn't change existing behavior

### CODE REVIEW

File: install/04_service.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ metadata:
     k8s-app: cluster-autoscaler-operator
   annotations:
     service.beta.openshift.io/serving-cert-secret-name: cluster-autoscaler-operator-cert
+    include.release.openshift.io/self-managed-high-availability: "true"
```
Comment (deads2k) [inline install/04_service.yaml:11]: alpha order please, throughout.

Comment (guillaumerose) [reply install/04_service.yaml]: done

**Review by michaelgugino — APPROVED** [review]
/lgtm

**Pull request merged**